packages/shopify-api/lib/auth/oauth/__tests__/create-session.test.ts

```diff
@@ -0,0 +1,157 @@
+import {createSession} from '../create-session';
+import {Session, shopifyApi} from '../../..';
+import {testConfig} from '../../../__tests__/test-config';
+
+let shop: string;
+const STATIC_UUID = 'test-uuid';
+
+jest.useFakeTimers().setSystemTime(new Date('2023-11-11'));
+
+jest.mock('uuid', () => ({v4: jest.fn(() => STATIC_UUID)}));
+jest.requireMock('uuid');
+
+beforeEach(() => {
+  shop = 'someshop.myshopify.io';
+});
+
+describe('createSession', () => {
+  describe('when receiving an offline token', () => {
+    [{isEmbeddedApp: true}, {isEmbeddedApp: false}].forEach(
+      ({isEmbeddedApp}) => {
+        test(`creates a new offline session when embedded is ${isEmbeddedApp}`, () => {
```

Comment:
We could use `test.each` here (untested):
```suggestion
  describe('when receiving an offline token', () => {
    test.each([true, false])(`creates a new offline session when embedded is %s`, (isEmbeddedApp) => {
```

Reply:
Updated the test 🙂